src/main/kotlin/racingcar/domain/car/Car.kt

```diff
@@ -0,0 +1,21 @@
+package racingcar.domain.car
+
+import racingcar.domain.moving.MoveStrategy
+import racingcar.domain.position.Position
+
+class Car(
+    private val moveStrategy: MoveStrategy
+) {
+    private var _position = Position(0)
+
+    val position: Position
+        get() = _position
```

Comment:
position을 생성자로부터 주입받게 만들어보는 것은 어떨까요?
Car를 생성할 때 초기 위치를 손쉽게 조정해줄 수 있는 장점을 지닙니다.

Reply:
또한, position을 내부에서는 변형할 수 있고 외부에서는 변형할 수 없게 만들고 싶다면 set만 private로 선언해주시면 됩니다. position객체가 불변 객체이기 때문에 이런 작성이 가능합니다🙂
```suggestion
    var position = Position(0)
    private set
```

Reply:
오 고민되는 지점이었는데, 완전 좋은 방법이네요 :) 감사합니다!